test/dotcom_web/views/helpers_test.exs

```diff
@@ -32,8 +32,8 @@ defmodule DotcomWeb.ViewHelpersTest do
     end
 
     test "non-formattable numbers don't get processed and don't become links" do
-      assert tel_link("0118 999 881 999 119 7253") ==
-               content_tag(:span, "0118 999 881 999 119 7253", [])
+      assert tel_link(nil) ==
```

Comment:
Shouldn't this be testing a string of numbers as per the title of the test?

Reply:
@kotva006 what happens to numbers that aren't formattable? is that accounted for somewhere else? as this isn't the same test.

Reply:
The behavior was [changed to support the tty numbers](https://github.com/mbta/dotcom/commit/b47c820dc3c448e03be4623d0a5c380f7878f438#diff-74dad5a6da9fddc99527eef73ae545c96084113a32757b13702548785166b885L41).
This allows us to support other numbers that aren't the 10 digit style (such as 711).  This test was never updated when the underlying logic was changed.

Reply:
Shouldn't we make it so that we *also* support TTY numbers instead of accepting anything?

```
iex(5)> DotcomWeb.ViewHelpers.tel_link("applesauce")
{:safe,
 [
   60,
   "a",
   [" class=\"", "no-wrap", 34, 32, "href", 61, 34, "tel:applesauce", 34],
   62,
   "applesauce",
   60,
   47,
   "a",
   62
 ]}
```

Reply:
I reworked the logic to support 3 digit or 11 and 10 digit numbers